tables.txt

```diff
@@ -7,7 +7,7 @@ CREATE TABLE report (
 );
 
 CREATE TABLE device_report (
-       device_id VARCHAR(50),
+       device_id VARCHAR(128),
```

Comment:
does this mean that the DB is going to need recreation?

Reply:
probably.  there's no data in it yet, though.